content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+यह पेज React के दस्तावेज और उससे सम्बंधित साधनो का ओवरव्यू है |
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** एक जावास्क्रिप्ट लाइब्रेरी है जो उपयोगकर्ता को इंटरफेसेस बनाने में काम आती है | [हमारे होमपेज](/) पर या [इस टुटोरिअल](/tutorial/tutorial.html) पर सीखें की React किस चीज़ के बारे में है |
 
----
-
-- [Try React](#try-react)
-- [Learn React](#learn-react)
-- [Staying Informed](#staying-informed)
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
-
-## Try React {#try-react}
+ ---
+ 
+ - [React का प्रयोग करके देखें](#try-react)
+ - [React सीखे](#learn-react)
+ - [सूचित रहे](#staying-informed)
+ - [वर्ज़न दस्तावेज़](#versioned-documentation)
+ - [क्या छूटा है?](#something-missing )
+ 
+## React का प्रयोग करके देखें {#try-react}
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need.** Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+React को धीरे-धीरे ग्रहण करने के लिए बनाया गया है, और  **आप जितनी कम और ज्यादा चाहे उतनी React इस्तेमाल कर सकते है |** यदि आप React का नमूना देखना चाहें, अपने सरल HTML पेज में कुछ इंटरएक्टिविट डालना चाहें, या एक जटिल React-संचालित ऐप बनाना चाहे, तो इसके लिए इस अनुभाग में दी गई लिंक्स आपकी सहायता करेंगी | 
 
-### Online Playgrounds {#online-playgrounds}
+### ऑनलाइन प्लेग्राउंड्स {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), or [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
+अगर आप React को इस्तेमाल करने में रूचि रखतें है, तो आप एक ऑनलाइन कोड प्लेग्राउंड इस्तेमाल कर सकते है | आप [CodePen](codepen://hello-world) या [CodeSandbox](https://codesandbox.io/s/new) पर एक Hello World टेम्पलेट इस्तेमाल करें |
 
-If you prefer to use your own text editor, you can also [download this HTML file](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html), edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so we'd only recommend using this for simple demos.
+अगर आप अपना खुद का Text Editor इस्तमाल करना चा रहे है, आप [इस HTML फाइल को डाउनलोड](https://raw.githubusercontent.com/reactjs/reactjs.org/master/static/html/single-file-example.html) कर सकतें हैं, एडिट करें, और अपने लोकल फाइलसिस्टम से इसे अपने ब्राउज़र में खोल लें, यह रनटाइम कोड ट्रांसफॉर्मेशन को धीमे करता है, हम इसे खाली सरल डेमो के लिए ही इस्तेमाल करने का अनुग्रह करते हैं |
```

Comment:
खुद का is unnecessary

Reply:
कर सकतें हैं| fir aap usse edit kar ke, aur aअपने लोकल फाइलसिस्टम से इसे अपने ब्राउज़र में खोल sakte hain. यह रनटाइम